TGM/src/main/java/network/warzone/tgm/command/CycleCommands.java

```diff
@@ -504,11 +501,27 @@ public static void team(CommandContext cmd, CommandSender sender) {
                 } else {
                     sender.sendMessage(ChatColor.RED + "/team size (team) (min) (max)");
                 }
+            } else if (cmd.getString(0).equalsIgnoreCase("shuffle")) {
+                List<MatchTeam> teams = TGM.get().getModule(TeamManagerModule.class).getTeams().stream().filter(matchTeam ->
```

Comment:
This could be simplified by using the [TeamManagerModule#getTeamsParticipating()](https://github.com/Warzone/TGM/blob/master/TGM/src/main/java/network/warzone/tgm/modules/team/TeamManagerModule.java#L205) method